.github/workflows/release-suite-message-system-config.yml

```diff
@@ -0,0 +1,44 @@
+name: "[Release] suite message system config"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-config:
+    if: ${{ github.repository == 'trezor/trezor-suite' && (github.ref == 'refs/heads/release-message-system-production' || github.ref == 'refs/heads/release-message-system-develop') }}
+    environment: ${{ github.ref == 'refs/heads/release-message-system-production' && 'production-message' || 'develop-message' }}
+    runs-on: ubuntu-latest
+    env:
+      RELEASE_ENV: ${{ github.ref == 'refs/heads/release-message-system-production' && 'production' || 'develop' }}
+      AWS_REGION: "eu-central-1"
+      AWS_CLOUDFRONT_ID: E1ERY5K2OTKKI1
+      ROLE_TO_ASSUME: ${{ github.ref == 'refs/heads/release-message-system-production' && 'arn:aws:iam::538326561891:role/gh_actions_suite_production_message' || 'arn:aws:iam::538326561891:role/gh_actions_suite_develop_message' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ env.ROLE_TO_ASSUME  }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Build and sign ${{ env.RELEASE_ENV }} message-system config file
```

Comment:
I dont like it like this, this way we would have to have secret for develop and produtction enviroment present and configured i think. Or maybe the string would be just empty sinc the secret would not be in the env secrets.

Reply:
Empty string would be ok...